### PR TITLE
Add theme to playground

### PR DESCRIPTION
**Pull request by msangtarash**

### CODE REVIEW

File: src/Bit.BlazorUIPlayground.Web/Pages/About.razor [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <PageTitle>About</PageTitle>
 
-<div class="page">
+<div class="page bit-css-clr-bg-primary">
```
Comment (msynk) [inline src/Bit.BlazorUIPlayground.Web/Pages/About.razor:5]: plz use the `BitCss` class instead

File: src/Bit.BlazorUIPlayground.Web/Pages/About.razor.css [hunk 2]
```diff
@@ -155,28 +154,29 @@
     box-sizing: border-box;
     color: #1A2027;
     transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
-    background-color: #FFFFFF;
+    background-color: var(--bit-clr-bg-primary);
```
Comment (msynk) [inline src/Bit.BlazorUIPlayground.Web/Pages/About.razor.css:157]: try not to use these css variables directly. instead, use the `BitCss` class to assign the required css class.

File: src/Bit.BlazorUIPlayground.Web/Shared/Header.razor [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 @code {
```
Comment (msynk) [inline src/Bit.BlazorUIPlayground.Web/Shared/Header.razor:3]: plz move this `@code` section to the end of the file and change the content based on what is implemented in the BlazorServer template project.

File: src/Bit.BlazorUIPlayground.Web/Shared/MainLayout.razor [hunk 4]
```diff
@@ -9,3 +9,18 @@
 
     <Footer />
 </main>
+
+@code {
```
Comment (msynk) [inline src/Bit.BlazorUIPlayground.Web/Shared/MainLayout.razor:13]: please move this code to the App.razor file.

File: src/Bit.BlazorUIPlayground.Web/Shared/MainLayout.razor [hunk 5]
```diff
@@ -9,3 +9,18 @@
...
+
+    [Inject] private IJSRuntime _js { get; set; }
+
+    protected override void OnAfterRender(bool firstRender)
+    {
+        if (firstRender)
+        {
+            BitThemeManager.init(_js);
+            var myTheme = new BitTheme();
+            BitThemeManager.ApplyBitTheme(myTheme);
```
Comment (msynk) [inline src/Bit.BlazorUIPlayground.Web/Shared/MainLayout.razor:23]: remove these 2 redundant lines of code. only the init call is required.

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Bit.BlazorUIPlayground.Web/Shared/MainLayout.razor (hunk 5 above)
Comment (ysmoradi) [reply src/Bit.BlazorUIPlayground.Web/Shared/MainLayout.razor:23]: Update to the latest pre release and use `Init` instead of `init`

**Review by msynk — APPROVED** [review]

**Pull request merged**